package.json

```diff
@@ -2,15 +2,16 @@
   "name": "vanilla-framework-react",
   "version": "0.1.3",
   "description": "Vanilla Framework, built using React",
-  "main": "build/index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "build": "node scripts/build.js",
+    "build": "",
     "build-storybook": "build-storybook",
     "clean": "rm -rf node_modules && rm -rf build",
     "lint": "eslint ./src/",
+    "prepublish": "export BABEL_ENV=production && rm -rf dist && babel src/lib -d dist",
```

Comment:
I think we should use `prepublishOnly` because `prepublish` ALSO runs before install which makes no logical sense to me.